src/com/spikes2212/command/drivetrains/TankDrivetrain.java

```diff
@@ -0,0 +1,77 @@
+package com.spikes2212.command.drivetrains;
+
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+
+/**
+ * This class represents a type of Drivetrain that its left and right sides are
+ * controlled independently, allowing it to move by giving each side a speed
+ * value separately.
+ *
+ * It can move forwards/backwards by giving both its sides an equal speed or
+ * turn by giving the sides different speeds
+ *
+ * @author Yuval Levy
+ */
+public class TankDrivetrain implements Subsystem {
```

Comment:
Make `TankDrivetrain` extend `SubsystemBase`. this is a class that implements `Subsystem` and also provides `Sendable` functionality.